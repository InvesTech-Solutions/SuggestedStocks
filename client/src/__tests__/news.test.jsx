import React from 'react';
import Newz from '../components/news.jsx';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

