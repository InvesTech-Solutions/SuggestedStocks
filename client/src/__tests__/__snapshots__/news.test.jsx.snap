// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    id="newsMainHead"
  >
    <h2>
      <strong>
         News 
      </strong>
    </h2>
    <div
      id="showMore"
    >
      <h6>
         Show More 
      </h6>
      <a
        href="#"
      />
    </div>
  </div>
  <div
    id="theNewz"
  >
    <Newz
      props={
        Object {
          "description": "Investing.com-Technology Stocks",
          "img": "http://lorempixel.com/400/200",
          "source": "Yahoo Finance",
          "time": "7h",
          "title": "Stocks, Stocks, and more Stocks",
          "url": "#",
          "viewership": 466,
        }
      }
    />
    <a
      href="https://www.google.com"
    />
  </div>
</div>
`;
